src/stavenote.ts

```diff
@@ -532,10 +532,10 @@ export class StaveNote extends StemmableNote {
         displaced,
         stemDirection,
         customGlyphCode: noteProps.code,
-        glyphFontScale: this.renderOptions.glyphFontScale,
-        xShift: noteProps.shiftRight,
-        stemUpXOffset: noteProps.stemUpXOffset,
-        stemDownXOffset: noteProps.stemDownXOffset,
+        // #FIXME: HACK to use 30 as fontSize default rather than 39 used with glyphs.
+        // HACK-BEGIN
+        glyphFontScale: (this.renderOptions.glyphFontScale / 39) * 30,
+        // HACK-END
```

Comment:
How will we resolve this hack later?

Reply:
The 39 / 30 will go away. This is needed for the extracted code.